app/views/products/_product.html.erb

```diff
@@ -4,6 +4,13 @@
       <%= image_tag product.avatar %>
     </div>
     <div class="caption">
+      <h6>
+        <% if product.calories != nil %>
```

Comment:
same here: use if without the `!= nil`, swap around the blocks, probably remove the else